tests/acceptance/features/bootstrap/WebDav.php

```diff
@@ -3624,20 +3700,18 @@ public function userOnHasDeletedFile(string $user, string $server, string $fileO
 	}
 
 	/**
-	 * @When user :user creates folder :destination using the WebDAV API
-	 *
 	 * @param string $user
 	 * @param string $destination
 	 * @param bool|null $isGivenStep
 	 *
-	 * @return void
+	 * @return ResponseInterface
 	 * @throws JsonException | GuzzleException
 	 * @throws GuzzleException | JsonException
 	 */
-	public function userCreatesFolder(string $user, string $destination, ?bool $isGivenStep = false):void {
+	public function userCreateFolder(string $user, string $destination, ?bool $isGivenStep = false):ResponseInterface {
```

Comment:
there should be function for this. see `createFolder`

Reply:
okay i haven't rebased.
Will have to rebase and resolve conflicts